src/design-tokens/css/base/reset.css

```diff
@@ -4,14 +4,13 @@
   box-sizing: border-box;
 }
 
-/* TODO-AH: figure out what these tokens should be */
 ::-moz-selection {
   /* Code for Firefox */
   color: var(--eds-theme-color-text-highlight-foreground);
-  background: var(--eds-theme-color-text-highlight-background);
+  background: var(--eds-theme-color-background-utility-default-low-emphasis);
```

Comment:
noticed this documentation / reset glitch (using undefined token), so setting a token to use in this context